Makefile

```diff
@@ -260,7 +265,8 @@ endif
 	docker run --rm -v $(CURDIR)/dist/bin:/go/bin:rw $(CALICO_BUILD) /bin/sh -c "\
 	  echo; echo calico-node-$(ARCH) -v;	 /go/bin/calico-node-$(ARCH) -v; \
 	"
-	docker build --pull -t $(NODE_IMAGE):latest-$(ARCH) . --build-arg BIRD_IMAGE=$(BIRD_IMAGE) --build-arg QEMU_IMAGE=$(CALICO_BUILD) --build-arg GIT_VERSION=$(GIT_VERSION) -f ./Dockerfile.$(ARCH)
+## --platform fixes an issue where FROM SCRATCH imports an amd64 layer causing an architecture problem in cross compiling.
+	docker build --pull -t $(NODE_IMAGE):latest --platform=linux/$(TARGET_PLATFORM) . --build-arg BIRD_IMAGE=$(BIRD_IMAGE) --build-arg QEMU_IMAGE=$(CALICO_BUILD) --build-arg GIT_VERSION=$(GIT_VERSION) -f ./Dockerfile.$(ARCH)
```

Comment:
All docker images will be tagged as latest.
`--platform` should resolve the architecture problem reported in [here](https://github.com/projectcalico/calico/issues/4256).